packages/app/vite.styles.config.js

```diff
@@ -7,7 +7,7 @@ export default defineConfig({
     lib: {
       entry: path.resolve('src/styles.ts'),
       formats: ['es'],
-      fileName: () => 'styles.js',
+      fileName: 'styles',
```

Comment:
The CSS output file uses the same name: `styles.css`. The JS output file, `styles.js` remains empty as before (and deleted with `rimram` when running the `build:css` script).